hazelcast/src/main/java/com/hazelcast/internal/util/executor/CompletableFutureTask.java

```diff
@@ -35,12 +36,12 @@ public class CompletableFutureTask<V> extends DeserializingCompletableFuture<V>
 
     private volatile Thread runner;
 
-    CompletableFutureTask(Callable<V> callable) {
+    CompletableFutureTask(@Nonnull Callable<V> callable) {
```

Comment:
Not all callers guarantee the null-safety. E.g,  `CachedExecutorServiceDelegate#sumbit`